ucsb_ds_capstone_projects_2021/projects/nri/update1.md

```diff
@@ -1,2 +1,140 @@
-# Update 1
-Project updates
+# NRI - Update 1
+January 31, 2020
+
+
+Helen Huang, Shivam Misra, Priya Padmanabhan, Surya Pugal,  Yixiao Yue 
+
+
+## Description of The Project
+
+The project seeks to examine a variant of Alzheimer's disease passed genetically and determine how far back the mutation/mutations responsible for said disorder trace back -- identifying the timeframe when this occurred can provide us key insights into whether or not the mutation was the result of potential selective advantage.
+
+
+## Background
+
+Listed below are the terminology and biology concepts that are fundamental to our project.
+
+**Inheritance of Genetic Variation**
+* Identical by descent (IBD) - IBD length is correlated with historical relationship
+* Identical by state (IBS)
+
+**Haplotype**
+* Combination of alleles at different markers along the same chromosome that are inherited together
+* Haplotype decay over time because of recombination
+
+**Linkage Equilibrium (LE) vs. Linkage Disequilibrium (LD)**
+* Linkage Equilibrium (LE) - Independent assortment, Recombination occurs
+* Linkage Disequilibrium (LD) - Coinheritance, Non-randomly inherited
+* Measures
+  * (LE) 0 <-- D' (Recombination) --> 1 (LD)
+  * (LE) 0 <-- r^2 (Correlation) --> 1 (LD)
+
+**Illumina NGS Workflow**
+* Library presentation
+* Cluster generation
+* Sequencing
+* Data analysis
+
+**Data Generation Process**
+* DNA
+* FASTQ - Contains all the information (the reads from all DNA fragments)
+* SAM (Sequence Alignment Mapping) - Align the results to the reference genome
+* VCF (Variant Call Format) - Get rid of all the identical parts and keep the variants info only
+
+**Global IBD**
+* Coefficient of Relatedness
+  * For a pair of individuals separated by m meiosis, the average portion of the genome shared identical by descent (IBD) owing to that shared ancestry is 2^(-m)
+* Portion IBD
+  * Parent-Offspring (OP): 1st degree, 50%
+  * Full siblings: 1st degree, 50%
+  * Half siblings: 2nd degree, 25%
+  * Avuncular: 2nd degree, 25%
+  * Cousins: 3rd degree, 12.5%
+
+**Local IBD**
+* IBD0: shared 0 allele
+* IBD1: shared 1 allele
+* IBD2: shared 2 alleles
+* The length of the shared IBD segments is approximately exponentially distributed with a mean of 100 m^(-1) cM
+
+**Trio Phasing (Haplotype)**
+* Problem - We do not have genomes from both parents of every individual of interest
+* Solutions
+  - Option 1: Add more relatives into the phasing algorithm
+  - Option 2: Use known haplotypes from human populations and change your approach to a supervised method
+
+## Data Files
```

Comment:
Really liked that you put the different files and put the descriptions about whether they were human readable or not